trailing_spaces.sublime-settings

```diff
@@ -24,6 +24,10 @@
     // Set to false to ignore trailing spaces on the edited line.
     "trailing_spaces_include_current_line" : true,
 
+    // By default, any lines in the Find Results and Build output are ignored
+    // Add scopes to this list if you need to ignore them.
+    "trailing_spaces_scope_ignore": ["text.find-in-files", "source.build_output"],
```

Comment:
Would it make sense to also add diff scope here and remove from the deprecated option?

Reply:
Actually the diff syntax is not ignored. I swear I've seen it ignored somewhere...